www/horj18/staticky_web/vyrobky.css

```diff
@@ -0,0 +1,51 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i');
```

Comment:
pockat, to kazda stranka bude mit vlastni css soubor?
neopakuje se v nich neco? v rámci jednoho webu by měly být všechny styly v jednom css souboru

Reply:
Opravím, nevím co mě to napadlo 😰